PayPal.xcodeproj/project.pbxproj

```diff
@@ -2359,7 +2359,7 @@
 				MARKETING_VERSION = 1.0;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				MTL_FAST_MATH = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = "com.paypal.ios-sdk.Card";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.paypal.ios-sdk.CardPayments";
```

Comment:
It looks like the debug bundle ID is still using `Card`:
<img width="891" alt="Screen Shot 2023-01-30 at 1 17 04 PM" src="https://user-images.githubusercontent.com/20733831/215573882-d68e0778-1a5b-458e-a813-a9270841f722.png">

Reply:
Same for `CardPaymentTests` though I imagine we want this to be `CardPaymentsTests` in both places instead of the singular `Payment`.

Reply:
i'll check again if i missed another spot, but is there a way for you to do a "clean"? In Android Studio sometimes we have to do an "Invalidate Cache" to really test stuff that affect the build. Is there an equivalent in xcode?

Reply:
@jaxdesmarais never mind!
i found were it was missing, thank you!